packages/i18n/src/locales/en.i18n.json

```
@@ -2540,6 +2540,7 @@
   "Install_rocket_chat_on_your_preferred_desktop_platform": "Install Rocket.Chat on your preferred desktop platform.",
   "Installation": "Installation",
   "Installed": "Installed",
+  "Private": "Private",
```

Biome (2.1.2)

[error] The key Private was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)